sharqiyya-app-homepage/index.html

```diff
@@ -0,0 +1,557 @@
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">14</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">15</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">16</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">17</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">18</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">19</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">20</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">21</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <img class="dates-bar__arrow" src="images/icons/forward-arrow-grey.svg">
+
+            </div>
+            <div class="dates-bar__right">
+              <img src="images/icons/search.svg">
+              <div class="dates-bar__search">Search for an event or activity</div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="homepage">
+        <div class="tags-bar">
+          <div class="tags-bar__tag tags-bar__tag_selected">
+            Everything
+          </div>
+          <div class="tags-bar__tag">
+            Halloween Special
+          </div>
+          <div class="tags-bar__tag">
+            Arts & Culture
+          </div>
+          <div class="tags-bar__tag">
+            Conferences & Workshops
+          </div>
+          <div class="tags-bar__tag">
+            Exhibitions
+          </div>
+          <div class="tags-bar__tag">
+            Sports & Outdoor
+          </div>
+          <div class="tags-bar__tag">
+            Food & Drinks
+          </div>
+          <div class="tags-bar__tag">
+            Nightlife
+          </div>
+          <div class="tags-bar__tag">
+            More
+          </div>
+        </div>
+
+        <div class="homepage-content">
+          <div class="homepage-content__title">
+            15 EVENTS FOUND ON SATURDAY AUGUST 9
+          </div>
+          <div class="homepage-content__events-list">
+
+
+            <div class="homepage-content__event">
```

Comment:
I would have gone with making the event cards their own block, like `event-card` and everything inside is an element of that. It also would help because they are reused in the event page.